docs/accounts/switching.md

```diff
@@ -0,0 +1,17 @@
+# Switching Between Multiple Accounts
+
+If you configure multiple accounts in K-9, the default view will be the Unified Inbox.
```

Comment:
I don't think that's true. Right now the Unified Inbox is always enabled.

Reply:
updated